.gitignore

```diff
@@ -56,6 +56,7 @@ cover
 .directory
 .gdb_history
 .DS_Store?
```

Comment:
Not sure if this one with a question mark is needed, but based on the fact I accidentally committed a few .DS_STORE without the question mark this needs to be added.